src/loaders/abstract-loader.coffee

```diff
@@ -306,6 +307,7 @@ class AbstractLoader
     options = @loadOptions
     syncOptions =
       data: {}
+      featureName: @loadOptions.feature_name
```

Comment:
Seems strange that this key is camel case whereas the other one is snake case

Reply:
`@loadOptions` are the interface when making a fetch. Those are snake case by practice since the idea is to mirror their Brainstem counterpart. The `syncOptions` is the internal JavaScript options object.

I agree that it is a little weird. Perhaps we should deprecate snake case all together unless it is user provided values (like values for filters and includes and what-have-yous).